[2024-12-07T02:36:10.027-0300] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-07T02:36:10.036-0300] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: hiring_lineal.preprocess_and_train manual__2024-12-07T05:30:45.798598+00:00 [queued]>
[2024-12-07T02:36:10.041-0300] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: hiring_lineal.preprocess_and_train manual__2024-12-07T05:30:45.798598+00:00 [queued]>
[2024-12-07T02:36:10.042-0300] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2024-12-07T02:36:10.096-0300] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): preprocess_and_train> on 2024-12-07 05:30:45.798598+00:00
[2024-12-07T02:36:10.103-0300] {standard_task_runner.py:72} INFO - Started process 38475 to run task
[2024-12-07T02:36:10.106-0300] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'hiring_lineal', 'preprocess_and_train', 'manual__2024-12-07T05:30:45.798598+00:00', '--job-id', '21', '--raw', '--subdir', 'DAGS_FOLDER/dag_lineal1.py', '--cfg-path', '/tmp/tmpaaid_lz7']
[2024-12-07T02:36:10.106-0300] {standard_task_runner.py:105} INFO - Job 21: Subtask preprocess_and_train
[2024-12-07T02:36:10.175-0300] {task_command.py:467} INFO - Running <TaskInstance: hiring_lineal.preprocess_and_train manual__2024-12-07T05:30:45.798598+00:00 [running]> on host crisu-VivoBook-S14-X411UF
[2024-12-07T02:36:10.301-0300] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='hiring_lineal' AIRFLOW_CTX_TASK_ID='preprocess_and_train' AIRFLOW_CTX_EXECUTION_DATE='2024-12-07T05:30:45.798598+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-07T05:30:45.798598+00:00'
[2024-12-07T02:36:10.302-0300] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-12-07T02:36:10.302-0300] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-12-07T02:36:10.302-0300] {logging_mixin.py:190} INFO - Current task name:preprocess_and_train state:running start_date:2024-12-07 05:36:10.036814+00:00
[2024-12-07T02:36:10.302-0300] {logging_mixin.py:190} INFO - Dag name:hiring_lineal and current dag run status:running
[2024-12-07T02:36:10.303-0300] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-07T02:36:10.304-0300] {taskinstance.py:3311} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/crisu/miniconda3/envs/lab13.1/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/crisu/miniconda3/envs/lab13.1/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
  File "/home/crisu/miniconda3/envs/lab13.1/lib/python3.9/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/home/crisu/miniconda3/envs/lab13.1/lib/python3.9/site-packages/airflow/models/baseoperator.py", line 417, in wrapper
    return func(self, *args, **kwargs)
  File "/home/crisu/miniconda3/envs/lab13.1/lib/python3.9/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
  File "/home/crisu/miniconda3/envs/lab13.1/lib/python3.9/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
  File "/home/crisu/miniconda3/envs/lab13.1/lib/python3.9/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/home/crisu/Universidad/MDS7202-Primavera2024/Lab 13 - Airflow/dags/hiring_functions.py", line 58, in preprocess_and_train
    train_data = pd.read_csv(train_path)
  File "/home/crisu/miniconda3/envs/lab13.1/lib/python3.9/site-packages/pandas/io/parsers/readers.py", line 1026, in read_csv
    return _read(filepath_or_buffer, kwds)
  File "/home/crisu/miniconda3/envs/lab13.1/lib/python3.9/site-packages/pandas/io/parsers/readers.py", line 620, in _read
    parser = TextFileReader(filepath_or_buffer, **kwds)
  File "/home/crisu/miniconda3/envs/lab13.1/lib/python3.9/site-packages/pandas/io/parsers/readers.py", line 1620, in __init__
    self._engine = self._make_engine(f, self.engine)
  File "/home/crisu/miniconda3/envs/lab13.1/lib/python3.9/site-packages/pandas/io/parsers/readers.py", line 1880, in _make_engine
    self.handles = get_handle(
  File "/home/crisu/miniconda3/envs/lab13.1/lib/python3.9/site-packages/pandas/io/common.py", line 873, in get_handle
    handle = open(
FileNotFoundError: [Errno 2] No such file or directory: '/home/crisu/Universidad/MDS7202-Primavera2024/Lab 13 - Airflow/dags/output_2024-12-07/splits/train.csv'
[2024-12-07T02:36:10.312-0300] {logging_mixin.py:190} INFO - Task instance in failure state
[2024-12-07T02:36:10.312-0300] {logging_mixin.py:190} INFO - Task start:2024-12-07 05:36:10.036814+00:00 end:2024-12-07 05:36:10.311977+00:00 duration:0.275163
[2024-12-07T02:36:10.313-0300] {logging_mixin.py:190} INFO - Task:<Task(PythonOperator): preprocess_and_train> dag:<DAG: hiring_lineal> dagrun:<DagRun hiring_lineal @ 2024-12-07 05:30:45.798598+00:00: manual__2024-12-07T05:30:45.798598+00:00, state:running, queued_at: 2024-12-07 05:30:45.808698+00:00. externally triggered: True>
[2024-12-07T02:36:10.313-0300] {logging_mixin.py:190} INFO - Failure caused by [Errno 2] No such file or directory: '/home/crisu/Universidad/MDS7202-Primavera2024/Lab 13 - Airflow/dags/output_2024-12-07/splits/train.csv'
[2024-12-07T02:36:10.313-0300] {taskinstance.py:1225} INFO - Marking task as UP_FOR_RETRY. dag_id=hiring_lineal, task_id=preprocess_and_train, run_id=manual__2024-12-07T05:30:45.798598+00:00, execution_date=20241207T053045, start_date=20241207T053610, end_date=20241207T053610
[2024-12-07T02:36:10.371-0300] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-07T02:36:10.371-0300] {standard_task_runner.py:124} ERROR - Failed to execute job 21 for task preprocess_and_train ([Errno 2] No such file or directory: '/home/crisu/Universidad/MDS7202-Primavera2024/Lab 13 - Airflow/dags/output_2024-12-07/splits/train.csv'; 38475)
Traceback (most recent call last):
  File "/home/crisu/miniconda3/envs/lab13.1/lib/python3.9/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
  File "/home/crisu/miniconda3/envs/lab13.1/lib/python3.9/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
  File "/home/crisu/miniconda3/envs/lab13.1/lib/python3.9/site-packages/airflow/utils/cli.py", line 116, in wrapper
    return f(*args, **kwargs)
  File "/home/crisu/miniconda3/envs/lab13.1/lib/python3.9/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
  File "/home/crisu/miniconda3/envs/lab13.1/lib/python3.9/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
  File "/home/crisu/miniconda3/envs/lab13.1/lib/python3.9/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
  File "/home/crisu/miniconda3/envs/lab13.1/lib/python3.9/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/crisu/miniconda3/envs/lab13.1/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 3005, in _run_raw_task
    return _run_raw_task(
  File "/home/crisu/miniconda3/envs/lab13.1/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 273, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/home/crisu/miniconda3/envs/lab13.1/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 3159, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
  File "/home/crisu/miniconda3/envs/lab13.1/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 3183, in _execute_task
    return _execute_task(self, context, task_orig)
  File "/home/crisu/miniconda3/envs/lab13.1/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/crisu/miniconda3/envs/lab13.1/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
  File "/home/crisu/miniconda3/envs/lab13.1/lib/python3.9/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/home/crisu/miniconda3/envs/lab13.1/lib/python3.9/site-packages/airflow/models/baseoperator.py", line 417, in wrapper
    return func(self, *args, **kwargs)
  File "/home/crisu/miniconda3/envs/lab13.1/lib/python3.9/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
  File "/home/crisu/miniconda3/envs/lab13.1/lib/python3.9/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
  File "/home/crisu/miniconda3/envs/lab13.1/lib/python3.9/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/home/crisu/Universidad/MDS7202-Primavera2024/Lab 13 - Airflow/dags/hiring_functions.py", line 58, in preprocess_and_train
    train_data = pd.read_csv(train_path)
  File "/home/crisu/miniconda3/envs/lab13.1/lib/python3.9/site-packages/pandas/io/parsers/readers.py", line 1026, in read_csv
    return _read(filepath_or_buffer, kwds)
  File "/home/crisu/miniconda3/envs/lab13.1/lib/python3.9/site-packages/pandas/io/parsers/readers.py", line 620, in _read
    parser = TextFileReader(filepath_or_buffer, **kwds)
  File "/home/crisu/miniconda3/envs/lab13.1/lib/python3.9/site-packages/pandas/io/parsers/readers.py", line 1620, in __init__
    self._engine = self._make_engine(f, self.engine)
  File "/home/crisu/miniconda3/envs/lab13.1/lib/python3.9/site-packages/pandas/io/parsers/readers.py", line 1880, in _make_engine
    self.handles = get_handle(
  File "/home/crisu/miniconda3/envs/lab13.1/lib/python3.9/site-packages/pandas/io/common.py", line 873, in get_handle
    handle = open(
FileNotFoundError: [Errno 2] No such file or directory: '/home/crisu/Universidad/MDS7202-Primavera2024/Lab 13 - Airflow/dags/output_2024-12-07/splits/train.csv'
[2024-12-07T02:36:10.399-0300] {local_task_job_runner.py:266} INFO - Task exited with return code 1
[2024-12-07T02:36:10.413-0300] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-07T02:36:10.414-0300] {local_task_job_runner.py:245} INFO - ::endgroup::
